server/cron.go

```diff
@@ -1,24 +0,0 @@
-package server
-
-import (
-	"os"
-
-	"github.com/robfig/cron/v3"
-)
-
-// SetupCronTasks run two functions every hour
-func (c *Server) SetupCronTasks() {
-	cron := cron.New()
-	var cadence = "0 * * * *"
-	if os.Getenv("ENV") == "production" {
-		cadence = "1 * * * *"
-	}
-	if _, err := cron.AddFunc(cadence, func() {
-		if err := c.rotateIssuers(); err != nil {
```

Comment:
TLV2 still uses the crontasks infra outlined here to refill the buffer of creds.  Talking with @Sneagan we think it would be best to disable this setupCronTasks method based on a passed in configuration.

Reply:
Will implement and update soon.

Reply:
I've made this feature configurable with an environment variable and updated the Ads implementation to use the variable appropriately. Omitting the variable should leave the existing behavior unchanged.